tests/Unit/Framework/PyppFundamentals.hpp

```diff
@@ -87,7 +87,10 @@ PyObject* make_py_tuple(const Args&... t) {
 ///\cond
 template <>
 struct ToPyObject<void, std::nullptr_t> {
-  static PyObject* convert() { return Py_None; }
+  static PyObject* convert() {
+    Py_INCREF(Py_None);
```

Comment:
you can if you wish combine these two lines by using `Py_RETURN_NONE;`

Reply:
I think this is clearer, actually. I don't love a function that returns something having only:
```cpp
PyObject* convert() {
  Py_RETURN_NONE;
}
```
I find the lack of an actual return statement makes it harder to read :shrug: But I don't dislike it enough to fight over it. Whatever you prefer :)

Reply:
I don't care either way as, starting in the not too distant future, I'd likely have to look up either one to understand what is going on...  

An argument for preferring `Py_RETURN_NONE` is that we could add a CI file test for not having `return Py_None;` to prevent a future bug...

Reply:
I like the idea of preventing this bug with a file check. FYI, this is the definition of `Py_RETURN_NONE`:

```c++
#define Py_RETURN_NONE return Py_NewRef(Py_None)
```
https://github.com/python/cpython/blob/8b6c4a921af6d5d0a9640211ac93d7886a55a8f3/Include/object.h#L563

Reply:
Yea, add a CI check is a good argument for using `Py_RETURN_NONE`. I'll adjust